pages/kids/pdf/[kidsDictionaryId].tsx

```diff
@@ -0,0 +1,115 @@
+/* eslint-disable @next/next/no-img-element */
+import React, { useMemo, FunctionComponent } from 'react';
+import { getCountryVariant } from 'utils/locales';
+import { GetStaticPaths, GetStaticProps, InferGetStaticPropsType } from 'next';
+import { getServerSideTranslations } from '../../../utils/localization';
+import { useLanguage } from '../../../utils/useLanguageHook';
+import { DictionaryDataObject, fetchDictionary, getKidsCategory, Phrase } from '../../../utils/getDataUtils';
+import { ParsedUrlQuery } from 'querystring';
+
+interface UrlParams extends ParsedUrlQuery {
+  kidsDictionaryId: string;
+}
+
+interface PhraseImageProps {
+  phrase: Phrase;
+}
+
+const KidsDictionaryPage = ({ dictionary }: InferGetStaticPropsType<typeof getStaticProps>): JSX.Element => {
+  const kidsCategory = useMemo(() => getKidsCategory(dictionary), [dictionary]);
+  const { currentLanguage, otherLanguage } = useLanguage();
+
+  const PhraseImage: FunctionComponent<PhraseImageProps> = ({ phrase }) => {
+    // NextImage does not work properly in PDFs, we use a regular <img> element instead
+    return kidsCategory?.translations ? (
+      /* eslint-disable-next-line @next/next/no-img-element */
+      <div className="h-[620px] overflow-hidden">
+        <img src={phrase.getImageUrl() ?? ''} width="600" className="mx-auto" height="600" alt={phrase.getTranslation(currentLanguage)} />
+      </div>
+    ) : null;
+  };
+
+  return (
+    <div>
+      <div className="flex flex-row">
+        <table className="text-xl font-medium basis-1/2">
+          <tbody>
+            {kidsCategory?.translations
+              .filter((_, index) => index % 2 === 0)
+              .map((phrase) => (
+                <tr key={phrase.getId()}>
+                  <td>
+                    <div className="border-black border-1 h-[840px] mb-2">
+                      <PhraseImage phrase={phrase} />
+                      <div className="p-2 h-[218px] border-t-black border-transparent border-1 flex flex-col items-center justify-center">
```

Comment:
Small visual detail: When text fits on one line, it is centered (I guess thanks to flexbox) but when the text wraps to the second line, it is left-aligned (I guess the div is full width then). Try applying `text-align: center` or something to make it always centered :) 
<img width="1092" alt="image" src="https://github.com/cesko-digital/movapp/assets/11249993/78114bc9-4e67-4fd6-a660-d50487f711cc">

Reply:
this one should be fixed with new commit